### DISCUSSION TITLE
Bugs that I faced in Revanced 17.22.46 non root.

### DISCUSSION THREAD

**Discussion by ronin08** [ReVanced/revanced-discussions#220] [General]
When I open Revanced (non-root v17.22.36) and go to Explore>Music. On the homepage with several posters of music videos( with slide option), it doesn't show the video title. But after playing few videos, it seems to be fine.
![IMG_20220613_210803](https://user-images.githubusercontent.com/75478274/173391344-e9146684-f0c2-4df2-99cf-4dcb93324a17.jpg)

Other bugs I encountered in Revanced 17.22.36

> slide to down to refresh on homepage bug is still there

> still seeing ads in shorts tab

> cast button visible. no option to hide it

Interestingly all these bugs are present in the last vanced(v17.03.38) too!

Other Suggestions:
While switching quality, there's a big popup everytime that "quality changed to this" which is very irritating. It should be disabled. There's also a popup when we dislike a video that "feedback" shared with creator. This should be removed too. These popups stay for 5-6 seconds.
![Screenshot_2022-06-13-21-21-48-851_app revanced android youtube](https://user-images.githubusercontent.com/75478274/173394034-95d7f8a3-6723-4c2a-967e-c425da7548aa.jpg)
![IMG_20220613_212400](https://user-images.githubusercontent.com/75478274/173394448-edc61ba1-41f3-4307-bf28-399f22f71611.jpg)

Other than this, it's super smooth. I hope to see these bugs fixed in the final version. Shout out too all the devs working on this.

**Comment by oSumAtrIX** [comment]
Confirm with regular youtube, if the popups are there as well

**Reply by ronin08** [reply]
can confirm they are present in the regular one too. but the pop up layout is different

**Comment by oSumAtrIX** [comment]
![Screenshot_2022-06-13-18-15-47-32_64ef5fc2000c1caa954c114bb372e1d5](https://user-images.githubusercontent.com/13122796/173398972-13d246b2-1179-4595-9da9-b7821736f9e9.jpg)

I am unable to reproduce. Try out disabling certain patches and confirm one of them causes the issue.

**Reply by ronin08** [reply]
i haven't added any new patches. just followed this:
https://www.paget96projects.com/guides/revanced-is-here-and-here-is-how-you-can-install-it
which I found on r/revancedapp.
Can you tell which specific arguments you used while building the app. And where can I find all the arguments(give link of that particular github section).

And what about other issues. Kindly look into those too.

**Reply by ronin08** [reply]
or if you can dm me the exact apk you are using. that would be nice 🤩

**Reply by oSumAtrIX** [reply]
Check revanced/revanced-documentation and the help channel on the ReVanced Discord Server

**Reply by Andrei137** [reply]
hello fellow revanced enjoyer. I recently got my apk too and I got the same question ,,Where can I find all the arguments". So with my little knowledge I did some ,,testing" I guess. So, u go to this page https://github.com/revanced/revanced-patches/releases, you install the latest Source Code, you extract the zip, you open the folder, go to src/main/kotlin/app/revanced/patches/youtube and there you have every arguments (find it at *(suggestive name of the patch)/patch) in .kt files which you can open with notepad and inside them you will find @Name and between parenthesis the exact argument. Oh, and the arguments used in that guide are all the currently available arguments.

**Reply by oSumAtrIX** [reply]
This is the wrong place to discuss that. Join the ReVanced Discord server for questions like that.

**Reply by ronin08** [reply]
@oSumAtrIX are you considering to remove the popups, slide to refresh bug, explore panel automatically hiding on home page, and shorts ads bug.
